app/presenters/batch_presenter.rb

```diff
@@ -10,6 +10,7 @@ def report
         caution: checks.each.count { |check| check.status == :caution },
         broken: checks.each.count { |check| check.status == :broken },
         pending: checks.each.count { |check| check.status == :pending },
+        danger: checks.each.count { |check| check.status == :danger },
```

Comment:
Does anything need to be added to the test suite to cover these?

Reply:
I've pushed up a commit now 👍